rmw_connextdds_common/src/common/rmw_network_flow_endpoints.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright 2021 Ericsson AB
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "rmw_connextdds/rmw_impl.hpp"
+
+extern "C"
+{
+#if RMW_CONNEXT_HAVE_GET_NETWORK_FLOW_ENDPOINTS
```

Comment:
Looks like a feature flag made it through the refactoring :)

Reply:
Good spotting! That one escaped my grep filter. I will push a new commit soon.